### ISSUE TITLE
 CSwapChainBase::QueuePresentAndWait() looped for 21 iterations without a present event. 

### ISSUE THREAD

**Issue by sercankd** [ValveSoftware/Dota-2#1051]
This cause micro stutters while playing the game. I am using latest driver from nvidia

**Comment by gdrewb-valve** [comment]
Can you post your full system information from Steam?

**Comment by sercankd** [comment]
Computer Information:
    Manufacturer:  Gigabyte Technology Co., Ltd.
    Model:  GA-MA770-DS3
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  AuthenticAMD
    CPU Brand:  AMD Athlon(tm) 64 X2 Dual Core Processor 5600+
    CPU Family:  0xf
    CPU Model:  0x6b
    CPU Stepping:  0x2
    CPU Type:  0x0
    Speed:  2913 Mhz
    2 logical processors
    2 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Unsupported
    SSE4a:  Unsupported
    SSE41:  Unsupported
    SSE42:  Unsupported
    AES:  Unsupported
    AVX:  Unsupported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    Windows 7 (64 bit)
    NTFS:  Supported
    Crypto Provider Codes:  Supported 311 0x0 0x0 0x0

Video Card:
    Driver:  NVIDIA GeForce 9600 GT

```
DirectX Driver Name:  nvd3dum.dll
Driver Version:  9.18.13.4198
DirectX Driver Version:  9.18.13.4198
Driver Date: 13 Sept 2016
OpenGL Version: 3.3
Desktop Color Depth: 32 bits per pixel
Monitor Refresh Rate: 60 Hz
DirectX Card: NVIDIA GeForce 9600 GT
VendorID:  0x10de
DeviceID:  0x622
Revision:  0xa1
Number of Monitors:  2
Number of Logical Video Cards:  2
No SLI or Crossfire Detected
Primary Display Resolution:  1920 x 1080
Desktop Resolution: 3360 x 1080
Primary Display Size: 26.65" x 15.00"  (30.55" diag)
                                        67.7cm x 38.1cm  (77.6cm diag)
Primary Bus: PCI Express 16x
Primary VRAM: 512 MB
Supported MSAA Modes:  2x 4x 8x 
```

Sound card:
    Audio device: Speakers (High Definition Audio

Memory:
    RAM:  4094 Mb

Miscellaneous:
    UI Language:  English
    Microphone:  Not set
    Steam Controller Cable and Base:  Not set
    Media Type:  Undetermined
    Total Hard Disk Space Available:  476985 Mb
    Largest Free Hard Disk Block:  78283 Mb
    OS Install Date: Oca 01 1970
    Game Controller: None detected
    VR Headset: None detected

**Comment by ramma** [comment]
> Windows 7 (64 bit)

Just fyi, this tracker is specific to Mac and Linux issues. There's a forum thread about the same issue [here](http://dev.dota2.com/showthread.php?t=179605), and a few other threads with less replies. Looks like making sure the "Use my monitor's current resolution" fixed the issue for most in that thread.

**Comment by gdrewb-valve** [comment]
As ramma mentioned this forum is for Linux/Mac porting bugs and does not handle Windows issues.  Posting in dev.dota2.com is the right place.

**Comment by SLAYER-CODE** [comment]
Same error, but mine was a bit different the first thing the console didn't show what it should show.

![image](https://user-images.githubusercontent.com/51754456/224779741-3c5b99b7-3b87-4a32-8637-b2fedf1455b8.png)

Inside the console I found this thread error and sure enough, the game paused and not only the game but the entire desktop environment, I use awesomeWM. but in the end I managed to solve it by changing the vulkan stalone for the vulkan pro using the configuration parameters.

`VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/amd_pro_icd64.json %command%`
